### PR TITLE
update playbook format

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
Ready 

## Related Issues
fixes: https://github.com/demisto/etc/issues/21681

## Description
Check if a description already exists for a task and if so don't override it with an empty value

### CODE REVIEW

File: demisto_sdk/commands/format/update_playbook.py [hunk 1]
```diff
@@ -25,6 +25,10 @@ def add_description(self):
             self.yml_data['description'] = ''
 
         for task_id, task in self.yml_data.get('tasks', {}).items():
+            if task['task'].get('description'):
+                import ipdb; ipdb.set_trace()
```
Comment (anara123) [inline demisto_sdk/commands/format/update_playbook.py]: why you need this line?

Comment (idovandijk) [inline demisto_sdk/commands/format/update_playbook.py]: Why do we need debugger here?

Comment (idovandijk) [reply demisto_sdk/commands/format/update_playbook.py]: Anar commented, removing mine

Comment (ronykoz) [reply demisto_sdk/commands/format/update_playbook.py]: my bad, removed it

**Pull request merged**